lerna.json

```diff
@@ -4,5 +4,5 @@
   ],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "1.4.1"
+  "version": "1.4.3-alpha.0"
```

Comment:
shouldn't it be 1.5.0?

Reply:
Ya thanks... I have updated the polkadot dependencies to 3.7.1 so will bump the minor version for alpha release

Reply:
also new features added